examples/mcp/09_auth_example_auth0.py

```
@@ -0,0 +1,141 @@
+
+
+async def verify_auth(request: Request) -> dict[str, Any]:
+    try:
+        import jwt
+
+        auth_header = request.headers.get("authorization", "")
+        if not auth_header.startswith("Bearer "):
+            raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail="Invalid authorization header",
+            )
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)